bgpd/bgpd.c

```diff
@@ -8111,3 +8264,37 @@ struct peer *peer_lookup_in_view(struct vty *vty, struct bgp *bgp,
 	return peer;
 }
 
+
+/* BGP peer flag manipulation.  */
+int bgp_peer_flag_set(struct peer *peer, int flag)
```

Comment:
just void?

Reply:
I would replace this function with the set macro everywhere.